docs/conf.py

```diff
@@ -0,0 +1,90 @@
+# -*- coding: utf-8 -*-
+
+import os
+import sys
+from pathlib import Path
+
+# -- Sphinx Options -----------------------------------------------------------
+
+# If your project needs a minimal Sphinx version, state it here.
+needs_sphinx = "3.0"
+
+extensions = [
+    "sphinx.ext.autodoc",
+    "sphinx.ext.extlinks",
+    "sphinx.ext.intersphinx",
+    "sphinx.ext.todo",
+    "sphinxarg.ext",  # Automatic argparse command line argument documentation
+]
+
+autodoc_default_options = {
+    "members": True,
+}
+
+# The suffix(es) of source filenames.
+source_suffix = {
+    ".rst": "restructuredtext",
+    # '.txt': 'markdown',
+    # '.md': 'markdown',
```

Comment:
Need to uncomment this line, add recommonmark as a dependency above, and in the requirements.txt file.
Otherwise, markdown sources won't be processed by Sphinx.

Reply:
That is not required if sources are converted to rst.